src/telliot_feeds/cli/utils.py

```diff
@@ -42,7 +69,7 @@ def build_feed_from_input() -> Optional[DataFeed[Any]]:
     """
     try:
         query_type = input("Enter a valid Query Type: ")
-        feed = DATAFEED_BUILDER_MAPPING[query_type]
+        feed: DataFeed[Any] = DATAFEED_BUILDER_MAPPING[query_type]
     except KeyError:
         click.echo(f"No corresponding datafeed found for Query Type: {query_type}\n")
         return None
```

Comment:
Your catch for ValueError on line 94 should only encompass line 84, where you attempt to cast `val` by `param_dtype`. This makes it easier for us to isolate errors in the future..

Reply:
good call!